CMSgov/onemac
# OY2 21351: Package View - Email Blast to State Submitters/State Sys Admins when Package is Withdrawn

Story: https://qmacbis.atlassian.net/browse/OY2-21351
Endpoint: See github-actions bot comment

### Details
When a package is withdrawn, ALL active submitting users for that territory should receive the email notification.

### Changes
- Added a database query to find the active state submitters and state system admin emails
- reconfigured the email sending to be able to handle the asynchronous query
- moved some functionality into the config for readability/ease of finding business requirements
- Added use of the GSI2 index on "user role items" to be able to query the active users for state submitters (couldn't use the contactInfo way we use for cms approvers because state submitters can have different statuses for different territories all connected to one email address)
- Updated the seed data
- Created a migrate script to handle data changes moving up the environments.

### Implementation Notes
- BRIEFLY considered an aggregate item to hold the active email list, updated as users are added and/or their statuses changed ... this *would* be an excellent practice use case for using the stream to update aggregates, BUT... it's also a major paradigm shift that I'm not sure we want to do just yet....

### Test Plan
1. Log in as a Submitting User with a valid email address
2. Navigate to Package Dashboard
3. Withdraw a package
4. Verify you got the Withdrawal Email Receipt.
